script.js

```diff
@@ -1,4 +1,299 @@
+//     return "Player Wins! " + compareHand();
+//   } else if (handTotal(playerHand) < handTotal(computerHand)) {
+//     return "Computer Wins! " + compareHand();
+//   } else {
+//     return "Both players have tied! " + compareHand();
+//   }
+// }
+
+function playerBust(playerHand) {
+  if (handTotal(playerHand) > 21) {
+    return "You're busted!";
+  } else {
+    return false;
+  }
+}
+
+function checkPlayerCards() {
+  var playerCards = `Player total is ${handTotal(
+    playerHand
+  )}. Player is holding `;
+  for (var j = 0; j < playerHand.length; j += 1) {
+    // cannot make a linebreak in statement
+    playerCards += playerHand[j].name + " of " + playerHand[j].suit + "; ";
+  }
+  return playerCards;
+}
+
+function checkBothBelow17() {
+  console.log("Player's Hands");
+  console.log(handTotal(playerHand));
+  console.log("Computer's Hands");
+  console.log(handTotal(computerHand));
+  if (handTotal(playerHand) < 17 || handTotal(computerHand) < 17) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function passTurn() {
+  computerDraws();
+  return checkPlayerCards();
+}
+
+function resetGame() {
+  var deck = [];
+  deck = makeDeck();
+  shuffledDeck = shuffleCards(deck);
+  console.log(shuffledDeck);
+  playerHand = [];
+  computerHand = [];
+  initializeCards();
+  console.log(playerHand);
+  console.log(computerHand);
+  return checkPlayerCards();
+}
+
+function makeDeck() {
+  var cardDeck = [];
+  var suitIndex = 0;
+  while (suitIndex < suits.length) {
+    var currentSuit = suits[suitIndex];
+    var rankCounter = 1;
+    while (rankCounter <= 13) {
+      var cardName = rankCounter;
+      if (cardName == 1) {
+        cardName = "Ace";
+      } else if (cardName == 11) {
+        cardName = "Jack";
+      } else if (cardName == 12) {
+        cardName = "Queen";
+      } else if (cardName == 13) {
+        cardName = "King";
+      }
+      var card = {
+        name: cardName,
+        suit: currentSuit,
+        rank: rankCounter,
+      };
+      cardDeck.push(card);
+      rankCounter += 1;
+    }
+    suitIndex += 1;
+  }
+  return cardDeck;
+}
+function getRandomIndex(max) {
+  return Math.floor(Math.random() * max);
+}
+
+function shuffleCards(cardDeck) {
+  var currentIndex = 0;
+  while (currentIndex < cardDeck.length) {
+    var randomIndex = getRandomIndex(cardDeck.length);
+    var randomCard = cardDeck[randomIndex];
+    var currentCard = cardDeck[currentIndex];
+    cardDeck[currentIndex] = randomCard;
+    cardDeck[randomIndex] = currentCard;
+    currentIndex = currentIndex + 1;
+  }
+  return cardDeck;
+}
+var deck = makeDeck();
+console.log(deck);
+var shuffledDeck = shuffleCards(deck);
+
+function initializeCards() {
+  for (var i = 0; i < 2; i += 1) {
+    playerHand.push(shuffledDeck.pop());
+    computerHand.push(shuffledDeck.pop());
+  }
+}
+
+initializeCards();
+console.log(playerHand);
+console.log(computerHand);
+
+function main(input) {
+  if (input.toLowerCase() == "stand") {
```

Comment:
If you notice you have to keep making the input lowercase, consider doing it at the start of the main function once. e.g.
var input = input.toLowerCase()